openedx/core/djangoapps/api_admin/migrations/0002_auto_20160325_1604.py

```diff
@@ -1,36 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Ah. I see you are squashing, perhaps in response to my earlier comment. I merely meant squashing the new migrations introduced by this work. There's no need to squash the existing migrations, too.